source-local/src/commonMain/kotlin/tachiyomi/source/local/io/Archive.kt

```diff
@@ -1,13 +1,17 @@
 package tachiyomi.source.local.io
 
 import com.hippo.unifile.UniFile
-import tachiyomi.core.common.storage.extension
 
 object Archive {
 
-    private val SUPPORTED_ARCHIVE_TYPES = listOf("zip", "cbz", "rar", "cbr", "epub")
+    private val SUPPORTED_ARCHIVE_TYPES = listOf("zip", "rar", "7z", "tar", "epub")
```

Comment:
`epub` already has a case in `Format.valueOf` seems to be legacy code.
```suggestion
    private val SUPPORTED_ARCHIVE_TYPES = listOf("zip", "rar", "7z", "tar")
```

Reply:
It's used here https://github.com/mihonapp/mihon/blob/36e40c099772d2cb53d4ec87b2b00f97fe455c98/source-local/src/androidMain/kotlin/tachiyomi/source/local/LocalSource.kt#L257

Reply:
Ok fair enough. Should probably be a separate check tho.

Reply:
https://github.com/mihonapp/mihon/blob/36e40c099772d2cb53d4ec87b2b00f97fe455c98/source-local/src/androidMain/kotlin/tachiyomi/source/local/LocalSource.kt#L188-L191
It's also used here, but I don't think epub will have ComicInfo.xml?

Reply:
It shouldn't under normal circumstances.